### DISCUSSION TITLE
The front-end page layout of GitHub's user history activity page is messed up

### DISCUSSION THREAD

**Discussion by CC11001100** [community/community#62437] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://github.com/limuen?tab=overview&from=2018-11-01&to=2018-11-30 

When I checked the homepage of this user, I found that the layout was messed up because the name of the warehouse was too long

![image](https://github.com/community/community/assets/12819457/755d6073-55e9-47ad-bee2-8d28d8648b8d)

![image](https://github.com/community/community/assets/12819457/47a21f51-24f3-4945-9f0c-92de605d291e)

**Comment by CC11001100** [comment]
steps: 

1. visit: https://github.com/limuen?tab=overview&from=2018-12-01&to=2018-12-31

2. click year 2018

**Comment by ossanna16** [comment]
Thank you for reporting this bug @CC11001100!

**Reply by CC11001100** [reply]
You're welcome. Hope GitHub gets better and better

**Reply by ossanna16** [reply]
Of course, we always strive to improve 🙂

**Comment by cehfisher** [comment]
Hi @CC11001100 - It looks like your issue was mistakenly added to the accessibility discussion board, but was resolved. @queenofcorgis could you please move this to the correct category and close it? Thanks!
